lib/ProductOpener/APIProductWrite.pm

```diff
@@ -196,7 +196,7 @@ Update packagings.
 
 =cut
 
-sub update_tags_fields ($request_ref, $product_ref, $field, $add_to_existing_tags, $tags_lc, $value) {
+sub update_tags_fields ($request_ref, $product_ref, $field, $add_to_existing_tags, $remove_tags, $tags_lc, $value) {
```

Comment:
If you change the parameters of a n existing function, you need to look for all the uses of that function to update the function calls to send the right parameters.

Reply:
Thanks, I made the changes but I'm having trouble generating a json file with the expected results. I'm running this command `make test-unit="api_v3_product_write.t --update-expected-results"`

Reply:
oooh, this is `make test-unit test="api_v3_product_write.t --update-expected-results"`

Reply:
Yes, I corrected the code but still 
I am going to try one more time

Reply:
@anaritadauane changed a bit the command (we have to add :: now because of yath the new  test runner)